Yoast/wordpress-seo
# Overwrites responsive styling for term pages

As the metabox has a restricted max-width on those pages we need to overwrite the styling for the help center video descriptions on those pages.

## Summary

This PR can be summarized in the following changelog entry:

* Adds more strict responsiveness styling for the video tabs in the Help Center

## Relevant technical choices:

* Added classes to the react components to allow overriding of default behaviour

## Test instructions

This PR can be tested by following these steps:

* Go to the taxonomy page and verify that the items are not placed next to each other for wide screens (1225px+)
* Go to the post edit page and verify that the items are not squished together - they will be placed below each other at screens smaller than 1480px
* Go to the other pages which have the Help Center and see that it still behaves as before/expected

Fixes #8039
